### ISSUE TITLE
Design bug: Preview takes focus *away* from terminal

### ISSUE THREAD

**Issue by EdmundsEcho** [iamcco/markdown-preview.nvim#686]
Thank you for this productive plugin.  

I have the plugin activate on `BufRead`.  When I fire up my markdown in Neovim, the browser opens up - Great!  __But__, in doing so, it takes the focus away from my terminal.  This behavior interrupts my workflow... and often has me (dare I say it) using the mouse to reactive my terminal! :))

Perhaps when the plugin fires-up the browser it does so without taking focus away from the terminal.

In case this isn't a universal problem, here are my specifics:
Browser: Firefox
OS: Mac 14.5

**Comment by EdmundsEcho** [comment]
If anyone has an idea how to control the focus in the source code of the plugin, I could try to introduce an option to configure the focus behavior.  I just need a pointer to get started.

**Comment by lthomas32** [comment]
Following, I love this app but I agree with you this does take away from my workflow productivity.

Browser: Firefox
OS: Pop_OS LTS 22

**Comment by Notaterritory** [comment]
I agree, I enabled the auto-preview function, but the immediate stealing of focus makes me want to turn it back off again.  It may be hard to do within the plugin, given the multiple terminals and operating systems used, but I wonder if there is a way to fix this with a custom command for opening the browser?

**Comment by Notaterritory** [comment]
@EdmundsEcho  - See the solution here
https://github.com/iamcco/markdown-preview.nvim/issues/185#issuecomment-2911789864
